Web/Edubase.Services/IntegrationEndPoints/OSPlaces/OSPlacesApiService.cs

```diff
@@ -25,8 +25,29 @@ public class OSPlacesApiService : IOSPlacesApiService
 
         private static Policy CreateRetryPolicy()
         {
-            var retryIntervalSettings = ConfigurationManager.AppSettings["OSPlacesApiServices_RetryIntervals"].Split(',');
-            var retryIntervals = retryIntervalSettings.Select(int.Parse).ToArray();
+            var defaultRetriesValue = new[] { 2 };
+            var retryEnabled = ConfigurationManager.AppSettings["OSPlacesApiServices_TimeoutIntervals_Enabled"] == "1";
+            var retryIntervalSettings = ConfigurationManager.AppSettings["OSPlacesApiServices_TimeoutIntervals"];
```

Comment:
This behaviour is specifically about the intervals between retry attempts? 

If so, I'd suggest not including `TimeoutIntervals` in the name as this implies it is specifically about the timeout (how long the request is allowed to run for, as opposed to the gaps between the requests).

Similarly, whether the retries are enabled or not?

Reply:
Probably should be RetryIntervals.  Does this allow an empty set of intervals as no retries at all?